suchvim/user-plugins.vim

```diff
@@ -3,12 +3,14 @@ let s:user_plugins = []
 function! SUCHVim_addPlugin(plugin)
     let plugin_command = "Plug '".a:plugin."'"
     call add(s:user_plugins, plugin_command)
+    return s:user_plugins
```

Comment:
Adding this line only to test the function might be a smell (the same goes for `SUCHVim_addPlugins`). Are you testing the right behavior in your test?